evergreen.yml

```
@@ -1988,3 +2075,4 @@ buildvariants:
       - name: release
       - name: snapshot
       - name: download-center-update
+      - name: publish-augmented-sbom
```

```suggestion
      - name: publish-augmented-sbom
      - name: download-center-update
```
Natacha wanted me to flip the order of these for the silk integration on JDBC.